src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,101 @@
+/* Outter Grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+@media (--tablet-and-down) {
+  .bleedBackground {
+    grid-row: 1;
+    grid-template-rows: auto;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
+  row-gap: 50px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+.gridAreaMiddle {
+  grid-column: 1 / span 12;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+
+  .gridAreaMiddle {
+    grid-column: 1 / span 13;
+  }
+}
+
+
+.innerWrapper {
+  background: var(--color-gray-300);
+  height: 100%;
```

Comment:
Since this is a bit tricky, I think it'd be useful to add a comment right after this that mentions that this makes all the slides the same height.

Reply:
👍 Added comment to address the reason we have 100% height there.